tests/unit/disco/components/TestAddon.js

```diff
@@ -386,25 +388,57 @@ describe(__filename, () => {
     });
   });
 
-  describe('<Addon type="theme"/>', () => {
-    let root;
+  describe.each([ADDON_TYPE_THEME, ADDON_TYPE_STATIC_THEME])(
+    `Addon for %s`,
+    async (type) => {
+      let root;
 
-    beforeEach(() => {
-      const data = { ...result, type: ADDON_TYPE_THEME };
-      root = renderAddon({ addon: data, ...data });
-    });
+      beforeEach(() => {
+        const data = { ...result, type };
+        root = renderAddon({ addon: data, ...data });
+      });
 
-    it('does render the theme image for a theme', () => {
-      expect(root.find('.theme-image')).toHaveLength(1);
-    });
+      it("doesn't render the logo for a theme", () => {
+        expect(root.find('.logo')).toHaveLength(0);
+      });
 
-    it("doesn't render the logo for a theme", () => {
-      expect(root.find('.logo')).toHaveLength(0);
+      it("doesn't render the description for a theme", () => {
+        expect(root.find('.editorial-description')).toHaveLength(0);
+      });
+    },
+  );
+
+  describe('<Addon type="statictheme"/>', () => {
+    const fullImage = 'https://addons.cdn.mozilla.net/full/54321.png';
+    const newAddonName = 'Summertime';
+
+    it('renders the correct image', () => {
+      const shallowRoot = renderAddon({
+        addon: {
+          ...result,
+          name: newAddonName,
+          previews: [
+            {
+              ...fakePreview,
+              image_url: fullImage,
+            },
+          ],
+          type: ADDON_TYPE_STATIC_THEME,
+        },
+      });
+      const image = shallowRoot.find('.Addon-theme-header-image');
+      expect(image.prop('src')).toEqual(fullImage);
+      expect(image).toHaveProp('alt', `Preview of ${newAddonName}`);
     });
+  });
 
+  describe('<Addon type="theme"/>', () => {
     it('calls installTheme on click', () => {
       const installTheme = sinon.stub();
-      const addon = result;
+      const addon = {
+        ...result,
+        type: ADDON_TYPE_THEME,
+      };
```

Comment:
What is `result` here? Also, I think we should be using `createInternalAddon()` from `core/reducers/addons`.

Reply:
results comes from `loadedAddons` function which is actually used in the DiscoPane component:

`const result = loadedAddons(_state)[0];`

Is that okay?

Reply:
ah ok, yes. sorry I was a bit lost but not your fault :)